attributes/haproxy.rb

```diff
@@ -41,23 +41,31 @@
   "user haproxy",
   "group haproxy",
   "tune.ssl.maxrecord 1419",
-  "spread-checks 5",
-  "# -- defaults settings section --",
-  "defaults",
+  "spread-checks 5"
+]
+
+default['haproxy']['default_config'] = [
   "mode http",
   "log global",
   "retries 3",
+  "",
   "# Options",
   "option httplog",
-node['haproxy']['logformat'],
+  if node['haproxy']['logging_json_enabled']
```

Comment:
is quite bad to have a conditional in an option list >.> . 
can the check be assigned outside of the variable list, so we can put just that variable in the list?
something like: `haproxy_logging =  node['haproxy']['logging_json_enabled'] ? node['haproxy']['json_logformat'] : node['haproxy']['logformat']` outside of the list , then add the variable `haproxy_logging` in the list